app/jobs/send_patient_otp_sms_job.rb

```diff
@@ -1,14 +1,17 @@
 class SendPatientOtpSmsJob < ApplicationJob
   queue_as :default
-  self.queue_adapter = :sidekiq
-
-  def perform(patient_authentication)
-    phone_number = patient_authentication.patient&.latest_mobile_number
 
+  def perform(passport_authentication)
+    phone_number = passport_authentication.patient&.latest_mobile_number
     return unless phone_number.present?
 
-    SmsNotificationService
-      .new(phone_number, ENV['TWILIO_PHONE_NUMBER'])
-      .send_patient_request_otp_sms(patient_authentication.otp)
+    NotificationService.new.send_sms(phone_number, otp_message(passport_authentication))
```

Comment:
I didn't do that because I saw some value in the future of being able to access the instance's twilio client, perhaps for debugging or logging.

Also, I don't personally like handing method-specific attributes to the initializer, when you can theoretically use the service instance to send a bunch of messages at once.